.eslintrc

```diff
@@ -30,13 +31,15 @@
   },
   "rules": {
     "indent": "off",
-    "indent-legacy": ["error", 2],
     "comma-dangle": ["error", "always-multiline"],
-    "jsx-quotes": ["error", "prefer-single"],
     "no-console": 0,
-    "quote-props": ["error", "consistent"],
     "react/jsx-filename-extension": 0,
-    "import/prefer-default-export": 0
+    "import/prefer-default-export": 0,
+    "react/prop-types": 0,
```

Comment:
Why are we turning off these?

Reply:
most of our components do not have prop types on them. since it lives inside `docs.js`